powershell-yaml.psm1

```diff
@@ -235,14 +235,17 @@ function ConvertFrom-Yaml {
     }
 }
 
+
 function ConvertTo-Yaml {
     [CmdletBinding()]
     Param(
-        [Parameter(Mandatory=$false,ValueFromPipeline=$true)]
+        [Parameter(ValueFromPipeline=$true)]
         [System.Object]$Data,
-        [Parameter(Mandatory=$false)]
+        
         [string]$OutFile,
-        [switch]$JsonCompatible=$false,
```

Comment:
Removing this without a deprecation warning first is not something I would like to do. If anyone is actually using this, their code will error out when they update. Please add this back, with a deprecation warning:

```powershell
[Obsolete("This parameter is obsolete and will be removed in the future.")]
```

Reply:
The JsonCompatible parameter can stay there